tanner/create_tables.py

```diff
@@ -0,0 +1,71 @@
+import asyncio
+
+
+class CreateTables:
+    @staticmethod
+    async def create_data_tables(pg_client):
+        """Create all the required tables in
+            the postgres database
+
+        Arguments:
+            pg_client {aiopg.pool.Pool}
+        """
+        async with pg_client.acquire() as conn:
+            async with conn.cursor() as cur:
+                await cur.execute(
+                    """
+                CREATE TABLE IF NOT EXISTS "paths" (
+                    "sess_uuid" UUID PRIMARY KEY NOT NULL UNIQUE,
+                    "path" TEXT NOT NULL,
+                    "timestamp" TIMESTAMP NOT NULL,
+                    "response_status" INT NOT NULL,
+                    "attack_type" INT NOT NULL
+                )
+                """
+                )
+
+                await cur.execute(
+                    """
+                CREATE TABLE IF NOT EXISTS "cookies" (
+                    "sess_uuid" UUID PRIMARY KEY NOT NULL UNIQUE,
+                    "key" TEXT NULL,
+                    "value" TEXT NULL
+                )
+                """
+                )
+
+                await cur.execute(
+                    """
+                CREATE TABLE IF NOT EXISTS "session_data" (
+                    "sess_uuid" UUID NOT NULL PRIMARY KEY,
+                    "snare_uuid" TEXT NOT NULL,
+                    "peer.ip" INET NOT NULL,
+                    "peer.port" INT NOT NULL,
+                    "location.country" TEXT NULL,
```

Comment:
@afeena how about putting location into a separate table?

Reply:
@glaslos I just worry that it might slow down the requests: if we store like this, instead of joining the tables, when we want to filter sessions by location, we just use simple `where`.
I don't think we need location info without session, what do you think?

Reply:
fine by me. I usually don't care too much about location information. We can still refactor once we feel like the sessions table is too busy